ember-bootstrap/addon/components/bs-modal.ts

```diff
@@ -639,17 +697,36 @@ export default class Modal extends Component {
    */
   @cached
   get scrollbarWidth() {
-    let scrollDiv = document.createElement('div');
+    const scrollDiv = document.createElement('div');
     scrollDiv.className = 'modal-scrollbar-measure';
-    let modalEl = this.modalElement;
+    const modalEl = this.modalElement;
+    if (!modalEl.parentNode) {
+      return 0;
+    }
     modalEl.parentNode.insertBefore(scrollDiv, modalEl.nextSibling);
-    let scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
-    scrollDiv.parentNode.removeChild(scrollDiv);
+    const scrollbarWidth = scrollDiv.offsetWidth - scrollDiv.clientWidth;
+    scrollDiv.parentNode?.removeChild(scrollDiv);
```

Comment:
Small leftover from previous refactoring.

```suggestion
    scrollDiv.parentNode.removeChild(scrollDiv);
```

Reply:
This one was actually not null-checked yet, so I've added the check for it as well.